tests/bundle/SystemInfo/Registry/IdentifierBasedTest.php

```diff
@@ -35,9 +33,9 @@ protected function setUp(): void
     /**
      * Test adding items to the registry, and getting items from it.
      *
-     * @covers \Ibexa\Bundle\SystemInfo\SystemInfo\Registry\IdentifierBased::getItem()
+     * @covers \IdentifierBased::getItem
```

Comment:
I think this is changed by accident. `IdentifierBased` is not a globally namespaced class.

In general, as per PHPUnit recommendations, `@covers` should only be used on test class level, not for particular methods. But I don't expect it "fixed" in this PR.